site/documentation/content/api/tenant/index.md

```diff
@@ -201,13 +201,25 @@ This allows for future *well-known* additions and also allows to add further inf
 
 The table below contains the properties which are used to configure a tenant's resource limits:
 
-| Name                     | Mandatory | JSON Type     | Default Value | Description |
-| :------------------------| :-------: | :------------ | :------------ | :---------- |
-| *max-connections*        | *no*      | *number*      | `-1`          | The maximum number of concurrent connections allowed from devices of this tenant. The default value `-1` indicates that no limit is set. |
-| *data-volume*            | *no*      | *object*      | `-`           | The maximum data volume allowed for the given tenant. Refer to  [Data Volume Configuration Format]({{< relref "#data-volume-configuration-format" >}}) for details.|
+| Name                | Mandatory | JSON Type     | Default Value | Description |
+| :------------------ | :-------: | :------------ | :------------ | :---------- |
+| *max-connections*   | *no*      | *number*      | `-1`          | The maximum number of concurrent connections allowed from devices of this tenant. The default value `-1` indicates that no limit is set. |
+| *max-ttl*           | *no*      | *number*      | `-1`          | The maximum time-to-live (in seconds) to use for events published by  devices of this tenant. Any default TTL value specified at either the tenant or device level will be limited to the max value specified here. If this property is set to a value other than `-1` and no default TTL is specified for a device, the max value will be used for events published by the device. A value of `-1` (the default) indicates that no limit is set. |
+| *data-volume*       | *no*      | *object*      | `-`           | The maximum data volume allowed for the given tenant. Refer to  [Data Volume Configuration Format]({{< relref "#data-volume-configuration-format" >}}) for details.|
 
 Protocol adapters SHOULD use the *max-connections* property to determine if a device's connection request should be accepted or rejected.
 
+Protocol adapters SHOULD use the *max-ttl* property to determine the *effective time-to-live* for events published by devices as follows:
+
+1. If a non-default *max-ttl* is set for the tenant, use that value as the *effective ttl*, otherwise set *effective ttl* to `-1`.
+1. If the event published by the device
+   1. contains a *ttl* property and *effective ttl* is not `-1` and the *ttl* value provided by the device is smaller than the
```

Comment:
Concerning the "event .. contains a ttl property" part: This might be confusing here because with #267, #266 still open, our protocol adapters don't support adopting a specific ttl for an event yet.

Reply:
The AMQP adapter does. The text is already complicated enough so I'd rather not make it even more complicated by adding another exception here ... I also do not think that people will have a problem figuring out which adapter supports this and which doesn't.